app/src/main/java/com/first/kotlin/MainActivity.kt

```diff
@@ -1,38 +1,26 @@
 package com.first.kotlin
 
+import android.content.Intent
 import android.os.Bundle
-import com.google.android.material.floatingactionbutton.FloatingActionButton
-import com.google.android.material.snackbar.Snackbar
+import android.widget.Button
 import androidx.appcompat.app.AppCompatActivity
-import android.view.Menu
-import android.view.MenuItem
 
 class MainActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+
         setSupportActionBar(findViewById(R.id.toolbar))
 
-        findViewById<FloatingActionButton>(R.id.fab).setOnClickListener { view ->
-            Snackbar.make(view, "Replace with your own action", Snackbar.LENGTH_LONG)
-                    .setAction("Action", null).show()
-        }
-    }
+        val button: Button =findViewById(R.id.button_first)
+        button.setOnClickListener{moveToMovieDetailsScreen()}
```

Comment:
Фигурные скобки принято выделять пробелами
```suggestion
        button.setOnClickListener { moveToMovieDetailsScreen() }
```

Reply:
Исправила